pages/blog/announcing-graphql-yoga-2.mdx

```diff
@@ -0,0 +1,278 @@
+
+No longer are you required to install dozens of dependencies to get features such as Subscriptions, File Uploads, CORS, Error Masking, and more.
+
+Building a GraphQL Yoga server requires a single import and only a few lines of code to start serving an API. And you also get GraphiQL for making development even easier.
+
+```jsx
+// 1. Import GraphQL Yoga
+import { createServer } from '@graphql-yoga/node'
+
+// 2. Create your server
+const server = createServer({
+  schema: {
+    typeDefs: /* GraphQL */ `
+      type Query {
+        hello: String
+      }
+    `,
+    resolvers: {
+      Query: {
+        hello: () => 'Hello Hello Hello'
+      }
+    }
+  }
+})
+
+// 3. Serve the API and GraphiQL
+server.start()
+```
+
+<p>&nbsp;</p>
+
+## The Yoga v2 experience
+
+### Your stack and habits
+
+The main goal of Yoga v2 is to **allow you to leverage all the GraphQL ecosystem** by being compatible with most of the existing schema-design, HTTP server libraries, and deployment environments.
+
+Built on top of a **modular and extendable GraphQL Server**, Yoga v2 allows you to use your preferred schema design approach and HTTP server library.
+
+For example, Yoga v2 is fully compatible with Express and Nexus, with no additional packages:
+
+```tsx
+import express from 'express'
+import { makeSchema, queryType } from 'nexus'
+import { createServer } from '@graphql-yoga/node'
+
+const Query = queryType({
+  definition(t) {
+    t.string('hello', { resolve: () => 'hello world!' })
+  }
+})
+const schema = makeSchema({
+  types: [Query]
+})
+
+const graphQLServer = createServer({ schema })
+
+const app = express()
+
+// Bind GraphQL Yoga to the `/graphql` endpoint
+// Here it takes the request and response objects and handles them internally
+app.use('/graphql', graphQLServer.requestListener)
+
+app.listen(4000, () => {
+  console.log('Running a GraphQL API server at http://localhost:4000/graphql')
+})
+```
+
+The same applies to **GraphQL Tools, Pothos, Nexus, TypeGraphQL, SDL first schema-design approaches, `graphql-js`, Apollo Tools, Fastify, Koa, Next.js SvelteKit, and Deno**.
+
+<br />
+
+Beyond the compatibility of schema-design and HTTP server libraries, Yoga v2 makes **deploying a GraphQL API to any environment seamless**.
+
+Here, a GraphQL API built with GraphQL Modules, deployed on Cloudflare Workers:
+
+```tsx
+import { createServer } from '@graphql-yoga/common'
+
+import { createApplication } from 'graphql-modules'
+import { helloWorldModule } from './helloWorld'
+
+const application = createApplication({
+  modules: [helloWorldModule]
+})
+
+const server = createServer({ schema: application.schema })
+
+self.addEventListener('fetch', (event: FetchEvent) => {
+  event.respondWith(server.handleRequest(event.request))
+})
+```
+
+Finally, Yoga v2 fits seamlessly in your GraphQL workflow and back-end ecosystem (Apollo Federation Gateway or Apollo Federation Service).
```

Comment:
I didn't fully understand that sentence.
Are you trying to say that Yoga can act as your Apollo Federation Gateway and also that it can act as a sub service?